src/integrationTest/resources/tests/cases/CaseControllerGetCaseTranscriptsTest/ignoreAutomaticTranscripts.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "hearing_date": "2023-01-01",
+    "type": "SENTENCING_REMARKS",
+    "requested_on": "2023-06-20T10:00:00Z",
```

Comment:
I think it would be better if each of these cases/hearings test data had different times, say minutes to make it more obvious that it is excluded

Reply:
good point, have fixed